tests/Integration/Symbol/Loader/FileSymbolLoaderTest.php

```diff
@@ -47,4 +48,16 @@ public function itFindsConsumedAutoloadFileRequireSymbols(): void
         self::assertArrayHasKey('Ds\Vector', $symbols);
         self::assertArrayHasKey('json_encode', $symbols);
     }
+
+    /**
+     * @test
+     */
+    public function itFindsArraySymbols(): void
+    {
+        /**
+         * TODO find a better strategy for testing this scenario
+         */
+        $this->expectNotToPerformAssertions();
+        $this->loadConsumedFileSymbols(self::ARRAY_NAMESPACE, [AutoloadType::PSR4]);
```

Comment:
I couldn't find a better way to test this. From what I could see, the `loadConsumerFileSymbols` does not return anything for this scenario. 

@icanhazstring can you advice on a better strategy to test this case?

Reply:
You should be able to do the same as here:
https://github.com/composer-unused/symbol-parser/blob/4dc2bf18ddf828c4932a30b2c1875e09c071fbc1/tests/Integration/Symbol/Loader/FileSymbolLoaderTest.php#L45

Just use two different symbols in your source files which you can test against.

Reply:
You should be able to do the same as here:
https://github.com/composer-unused/symbol-parser/blob/4dc2bf18ddf828c4932a30b2c1875e09c071fbc1/tests/Integration/Symbol/Loader/FileSymbolLoaderTest.php#L45

Just use two different symbols in your source files which you can test against.

Reply:
I gave it a quick run, but it won't load the PSR-4 symbols I defined in the `tests/assets/TestProjects/ArrayNamespace/composer.json`.

Reply:
That is because it parses consumed symbols from foreign dependencies. E.g. Of you are using a class or function from `ext-ds` it would be shown.

Reply:
I tried changing it to `AutoloadType::FILES` but it didn't work.

This however is bringing me to another question: Wouldn't the PSR namespaces loading be handled by the `PsrSymbolLoader` class instead of the `FileSymbolLoader`?

What do you think?

Here's a screenshot attached of the point in which the `FileSymbolLoader` is retrieved.
![image](https://user-images.githubusercontent.com/15325886/154114775-ed894b3a-30c9-4258-b2c5-d992893c445d.png)